src/libecalc/core/consumers/consumer_system.py

```diff
@@ -47,8 +47,14 @@ def get_subset_for_timestep(self, current_timestep: datetime) -> Self:
         ...
 
 
+class SystemCrossoverStream(Protocol):
+    stream_name: Optional[str]
```

Comment:
Burde vi tvinge navn? Er evnt fallbacken en autogenerering av `f{from}_{to}` eller lignende?

Reply:
Fallback er indeks, eller at man ikke trenger å referere til den streamen direkte (man mikser alle inlet streams). Kan alltids generere et navn for output. 

Kan vurdere å tvinge navn.